From 6a987751314f0a966aa526c744be757713fc4d6f Mon Sep 17 00:00:00 2001
From: Nicolas Le Bayon <nicolas.le.bayon@st.com>
Date: Thu, 26 Jan 2023 16:49:18 +0100
Subject: [PATCH 270/841] feat(stm32mp2): implement is_wakeup_from_standby
 service

This function, already implemented on STM32MP1 platform, is empty-added
on STM32MP2. Its prototype is already present in common part.

Upstream-Status: Inappropriate [ST downstream https://github.com/STMicroelectronics/arm-trusted-firmware.git 0b7ab5b9e40a128488f0b867d15db13a1546a515 feat(stm32mp2): implement is_wakeup_from_standby service]
Signed-off-by: Nicolas Le Bayon <nicolas.le.bayon@st.com>
Change-Id: Ie7de9a9f6477f8158e144f6626070a77fdc53ceb
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/tf-a/+/286974
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
Reviewed-by: Yann GAUTIER <yann.gautier@foss.st.com>
Domain-Review: Yann GAUTIER <yann.gautier@foss.st.com>
---
 plat/st/stm32mp2/stm32mp2_private.c | 6 ++++++
 1 file changed, 6 insertions(+)

diff --git a/plat/st/stm32mp2/stm32mp2_private.c b/plat/st/stm32mp2/stm32mp2_private.c
index 87f283730..935925dcd 100644
--- a/plat/st/stm32mp2/stm32mp2_private.c
+++ b/plat/st/stm32mp2/stm32mp2_private.c
@@ -413,6 +413,12 @@ bool stm32mp_skip_boot_device_after_standby(void)
 	return false;
 }
 
+bool stm32mp_is_wakeup_from_standby(void)
+{
+	/* TODO add source code to determine if platform is waking up from standby mode */
+	return false;
+}
+
 int stm32_risaf_get_instance(uintptr_t base)
 {
 	switch (base) {
-- 
2.47.2

